index.html

```diff
@@ -126,7 +126,7 @@ <h2 class="card-headline">Profesional Art Printing Data</h2>
       <div class="cards card-background-image card-background-image-lg2 pos-absolute-lg">
```

Comment:
- [ ] Your page title includes an underscore `_`. I think you could update to something more descriptive and professional for the users (e.g., Your name's Portfolio) :wink:
![image](https://user-images.githubusercontent.com/81522435/144895336-d8424658-cce2-4c77-8491-5fbd5c043134.png)

- [ ] You have done a good job with the accessibility :smiley: but I would suggest updating the `<hX>` tag used in the title "My Recent Works" and "About Me" sections because for good practices there can only be one `<h1>` per page. Since these are section titles, I think you could use `<h2>` tags :wink:
- [ ] I would suggest adding the `alt` attribute for your images so the screen readers can describe the image when necessary :smiley: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/144894859-5d474568-bc09-4d69-a2f7-986a154e86bf.png)
Please apply those suggestions when required for all the `<img>` :pushpin:

Reply:
Thanks 👏👏 
I've corrected and modified those area. 👌